apprise/plugins/NotifyMQTT.py

```diff
@@ -534,3 +517,38 @@ def parse_url(url):
 
         # return results
         return results
+
+    @property
```

Comment:
This is great, but if you `grep` for `pki` (for example), this list is used in several locations. The Todo in place was more to move this into the `utils.py' or maybe a new file called `secure.py`. A general reuse locating basically.

It would allow reuse. I need to verify how many other spots there are though. I think it was used for `xmpp` too, but that was dropped for now. I still like the way you massaged it.

Reply:
> The todo in place [...]

I did not have a look, but I added it with one of my recent commits.

> If you grep for pki (for example), this list is used in several locations.

I just tried, but did not find any other occurrances.

> I think it was used for `xmpp` too, but that was dropped for now.

Ah. That makes sense!